src/@types/output.ts

```diff
@@ -1,4 +1,9 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
+
+/**
+ * The objactive of every provider is to match this interface after parsing
```

Comment:
I guess this is typo 🫣
```suggestion
 * The objective of every provider is to match this interface after parsing
```